src/test/java/org/xstefank/TestUtils.java

```diff
@@ -0,0 +1,53 @@
+package org.xstefank;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import org.xstefank.model.yaml.FormatConfig;
+
+import java.io.File;
+import java.io.IOException;
+
+public class TestUtils {
+
+    public static final String YAML_DIR = "yaml";
+    public static final String JSON_DIR = "json";
+
+    public static final JsonNode TEST_PAYLOAD = loadJson(JSON_DIR + "/testPayload.json");
+    public static final JsonNode BAD_TEST_PAYLOAD = loadJson(JSON_DIR + "/badTestPayload.json");
+    public static final JsonNode EMPTY_PAYLOAD = getEmptyJsonPayload();
+    public static final FormatConfig FOMAT_CONFIG = getFormatConfig(YAML_DIR + "/testTemplate.yaml");
+
+    public static FormatConfig loadFormatFromYamlFile(String fileDir) throws IOException {
+        File file = new File(TestUtils.class.getClassLoader().getResource(fileDir).getFile());
+        return new ObjectMapper(new YAMLFactory()).readValue(file, FormatConfig.class);
+    }
+
+    private static JsonNode loadJson(String fileDir) {
+        try {
+            File file = new File(TestUtils.class.getClassLoader().getResource(fileDir).getFile());
+            return new ObjectMapper().readTree(file);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    private static JsonNode getEmptyJsonPayload() {
+        try {
+            return new ObjectMapper().readTree("");
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    private static FormatConfig getFormatConfig(String fileDir) {
```

Comment:
this method is kind of unnecessary as it only wraps the loadFormatFromYamlFile and prints exception like in other load methods in this class